src/vs/workbench/contrib/chat/browser/chat.contribution.ts

```diff
@@ -216,19 +216,19 @@ configurationRegistry.registerConfiguration({
 			default: true,
 			description: nls.localize('chat.sendElementsToChat.enabled', "Controls whether elements can be sent to chat from the Simple Browser."),
 			type: 'boolean',
-			tags: ['preview']
+			tags: ['experimental']
 		},
 		'chat.sendElementsToChat.attachCSS': {
```

Comment:
`attachCSS` seems useful for our debugging but I don't see why a user would want to set this?